when using libxcb.a static library also link with libXau.a

fix python error:
    File "/usr/lib/python3.8/site-packages/PIL/Image.py", line 94, in <module>
  ImportError: /usr/lib/python3.8/site-packages/PIL/_imaging.so: undefined symbol: XauGetBestAuthByAddr
                                                   
--- a/setup.py	2021-05-08 22:11:23.172946200 +0200
+++ b/setup.py	2021-05-08 22:11:53.482130861 +0200
@@ -730,6 +730,7 @@ class pil_build_ext(build_ext):
                 defs.append(("USE_WIN32_FILEIO", None))
         if feature.xcb:
             libs.append(feature.xcb)
+            libs.append('Xau')
             defs.append(("HAVE_XCB", None))
         if sys.platform == "win32":
             libs.extend(["kernel32", "user32", "gdi32"])
